entity-service-impl/src/test/java/org/hypertrace/entity/query/service/converter/filter/NullFilteringExpressionConverterTest.java

```diff
@@ -0,0 +1,129 @@
+package org.hypertrace.entity.query.service.converter.filter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.Optional;
+import org.hypertrace.core.documentstore.expression.impl.ConstantExpression;
+import org.hypertrace.core.documentstore.expression.impl.IdentifierExpression;
+import org.hypertrace.core.documentstore.expression.impl.LogicalExpression;
+import org.hypertrace.core.documentstore.expression.impl.RelationalExpression;
+import org.hypertrace.core.documentstore.expression.operators.LogicalOperator;
+import org.hypertrace.core.documentstore.expression.operators.RelationalOperator;
+import org.hypertrace.core.documentstore.expression.type.FilterTypeExpression;
+import org.hypertrace.core.grpcutils.context.RequestContext;
+import org.hypertrace.entity.query.service.EntityAttributeMapping;
+import org.hypertrace.entity.query.service.converter.ConversionException;
+import org.hypertrace.entity.query.service.converter.Converter;
+import org.hypertrace.entity.query.service.converter.identifier.IdentifierConverter;
+import org.hypertrace.entity.query.service.converter.identifier.IdentifierConverterFactory;
+import org.hypertrace.entity.query.service.v1.ColumnIdentifier;
+import org.hypertrace.entity.query.service.v1.LiteralConstant;
+import org.hypertrace.entity.query.service.v1.Operator;
+import org.hypertrace.entity.query.service.v1.Value;
+import org.hypertrace.entity.query.service.v1.ValueType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+class NullFilteringExpressionConverterTest {
+  private static final RequestContext REQUEST_CONTEXT = RequestContext.forTenantId("tenant1");
+
+  @Mock private EntityAttributeMapping entityAttributeMapping;
+  @Mock private IdentifierConverterFactory identifierConverterFactory;
+  @Mock private Converter<LiteralConstant, ConstantExpression> constantExpressionConverter;
+
+  private NullFilteringExpressionConverter nullFilteringExpressionConverter;
+
+  @BeforeEach
+  void setup() {
+    MockitoAnnotations.initMocks(this);
```

Comment:
nit: use Mockito extension instead. Not only it do this automatically, but it sets other defaults that we want like strict mode too.